pom.xml

```diff
@@ -419,6 +421,31 @@
   </reporting>
 
   <profiles>
+    <profile>
+      <id>slf4j2x-test</id>
```

Comment:
Perhaps this is answered previously but why do we need to add a profile in the parent module if a same profile is added in the child's module?

Reply:
I am still figuring this out, but the `maven-compiler-plugin` testInclude needs to be in the parent pom, otherwise compile would fail. My understanding is test classes from other modules (`cab-token-generator` in this case) tries to compile, and referencing to oauth classes that is not compiled from the module.

Reply:
Could we at least remove the `testIncludes` in either parent or child module?